### PR TITLE
change mockito inline to mockito core

### PR DESCRIPTION
**Pull request by anwilkie**
# What and why?
Testing out changing mockito inline to mockito core.

# How to test?
Deploy to environment, run acceptance tests, and test uploading sample file.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-803)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,10 +95,15 @@
         https://dev.to/scottshipp/how-to-fix-a-mockito-cannot-mock-this-class-exception-in-a-spring-boot-app-457e -->
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-inline</artifactId>
-            <version>4.6.1</version>
+            <artifactId>mockito-core</artifactId>
+            <version>5.5.0</version>
```
Comment (JacobHarrisonONS) [inline pom.xml:99]: the indentation looks a little off. can you fix it please

**Review by matthew-robinson-ons — COMMENTED** [review]
Functionally all looks good, happy to approve once comment above has been addressed

File: pom.xml (hunk 1 above)
Comment (anwilkie) [reply pom.xml]: Done.

**Review by JacobHarrisonONS — APPROVED** [review]
Reviewed and approved originally by Matthew. my comment about indentation is fixed

**Pull request merged**